share/elections/2025-02/spritzgebaeck.md

```diff
@@ -0,0 +1,21 @@
+# Sebastian Rakel (Spritzgebaeck)
+
+## About me
+My name is Sebastian Rakel, I'm a german Cloud Engineer working for a little company named . 
```

Comment:
Named `.`? Looks like this and the next sentence should have been combined :)

Reply:
oh thanks